src/create.js

```diff
@@ -32,6 +32,9 @@ const createEnv = async function() {
         'services': {
             'memcached': {
                 'image': images['memcached'],
+			},
+			'redis': {
```

Comment:
Can you fix the formatting here @jamesmorrison ?

Reply:
@christianc1 Sorry about that, sorted now 😄